tests/test_main.py

```diff
@@ -1,6 +1,136 @@
+    assert person_inst.name == name, (
+        f"Attribute 'name' for Person instance for {person} "
+        f"should equal to {name}"
+    )
+    assert person_inst.age == age, (
+        f"Attribute 'age' for Person instance for {person} "
+        f"should equal to {age}"
+    )
+
+
+def test_make_a_person_all_persons():
+    people = [
+        {'name': 'Ross', 'age': 30, 'wife': 'Rachel'},
+        {'name': 'Joey', 'age': 29, 'wife': None},
+        {'name': 'Phoebe', 'age': 31, 'husband': None},
+        {'name': 'Chandler', 'age': 30, 'wife': 'Monica'},
+        {'name': 'Monica', 'age': 32, 'husband': 'Chandler'},
+        {'name': 'Rachel', 'age': 28, 'husband': 'Ross'},
+    ]
+    person_list = make_a_person(people)
+    assert all(isinstance(person, Person) for person in person_list), (
+        "All elements in result of 'make_a_person' should be instance "
+        "of Person class"
+    )
+    assert len(person_list) == len(people), (
+        "Length of initial list should equal to length of function result"
+    )
+
+
+def test_make_a_person_order():
+    people = [
+        {'name': 'Ross', 'age': 30, 'wife': 'Rachel'},
+        {'name': 'Joey', 'age': 29, 'wife': None},
+        {'name': 'Phoebe', 'age': 31, 'husband': None},
+        {'name': 'Chandler', 'age': 30, 'wife': 'Monica'},
+        {'name': 'Monica', 'age': 32, 'husband': 'Chandler'},
+        {'name': 'Rachel', 'age': 28, 'husband': 'Ross'},
+    ]
+    person_list = make_a_person(people)
+    assert [person_dict['name'] for person_dict in people] == [person.name for person in person_list], (
+        "Order in function result should be the same"
+    )
+
+
+def test_make_a_person_has_wife():
+    people = [
+        {'name': 'Ross', 'age': 30, 'wife': 'Rachel'},
+        {'name': 'Joey', 'age': 29, 'wife': None},
+        {'name': 'Phoebe', 'age': 31, 'husband': None},
+        {'name': 'Chandler', 'age': 30, 'wife': 'Monica'},
+        {'name': 'Monica', 'age': 32, 'husband': 'Chandler'},
+        {'name': 'Rachel', 'age': 28, 'husband': 'Ross'},
+    ]
+    person_list = make_a_person(people)
+    assert hasattr(person_list[0], 'wife'), (
+        f"For 'people' equals to {people} "
+        f"Person with 'name' {person_list[0].name} should have "
+        f"attribute 'wife' with name {person_list[0].wife.name}"
+    )
+
+
+def test_make_a_person_has_wife_and_wife_have_husband():
+    people = [
+        {'name': 'Ross', 'age': 30, 'wife': 'Rachel'},
+        {'name': 'Joey', 'age': 29, 'wife': None},
+        {'name': 'Phoebe', 'age': 31, 'husband': None},
+        {'name': 'Chandler', 'age': 30, 'wife': 'Monica'},
+        {'name': 'Monica', 'age': 32, 'husband': 'Chandler'},
+        {'name': 'Rachel', 'age': 28, 'husband': 'Ross'},
+    ]
+    person_list = make_a_person(people)
+    assert hasattr(person_list[0], 'wife') and person_list[0].wife.husband == person_list[0], (
+        f"For 'people' equals to {people} "
```

Comment:
Also, if you use fixtures correctly - you don't need to add such line: `f"For 'people' equals to {people} "` - because people_data will be automatically shown in the console